std/net/isemail.d

```diff
@@ -1752,12 +1752,12 @@ enum AsciiToken
  */
 T max (T) (T[] arr)
 {
-    import std.algorithm/* : max*/;
+    static import std.algorithm.comparison;
```

Comment:
My goal for this PR is to only change the import code. There's a lot in these modules that needs to be fixed; I saw at least 15 package wide std.algorithm imports.